src/styles/index.scss

```diff
@@ -1,3 +1,170 @@
+$color-blue: #00acdc;
+$color-white: #fff;
+$color-dark: #060b35;
+$color-second-white: #f3f3f3;
+$font-width: 400;
+$font-family: "Roboto", sans-serif;
+$font-size-first: 10px;
+$font-size-second: 12px;
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.Imac {
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars {
+  display: flex;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: $color-blue;
+  border: 0;
+  margin-top: 16px;
+}
+
+button {
+  padding: 0;
+}
+
+.button {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  color: $color-white;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.button:hover {
+  border: 1px solid $color-blue;
+  border-radius: 5px;
+  color: $color-blue;
+  background-color: $color-white;
+  cursor: pointer;
+}
+
+.imacBackground {
+  width: 200px;
+  height: 408px;
+  background-color: $color-white;
+  border: 1px solid $color-second-white;
+  border-radius: 5px;
+  background-repeat: no-repeat;
+}
+
+.productCode {
+  line-height: 14px;
+  margin-top: 4px;
+  font-family: $font-family;
+  font-weight: $font-width;
+  font-size: $font-size-first;
+  color: #616070;
+}
+
+.price {
+  align-items: flex-end;
+  width: 31px;
+  line-height: 18px;
+  font-family: $font-family;
+  font-style: normal;
+  font-weight: $font-width;
+  font-size: $font-size-second;
+  color: #616070;
+  margin-right: 85px;
+}
+
+.money {
+  align-items: flex-end;
+  text-align: right;
+  width: 50px;
+  line-height: 18px;
+  font-family: $font-family;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  color: $color-dark;
+}
+
+.review {
+  text-align: right;
+  align-items: center;
+  line-height: 14px;
+  font-family: $font-family;
+  font-style: normal;
+  font-weight: $font-width;
+  font-size: $font-size-first;
+  color: $color-dark;
+}
+
+.text {
```

Comment:
Class name .text should follow the BEM naming convention and be more descriptive, like .card__product-name.